docs/running-e3sm-guide/guide-long-term-archiving.md

```diff
@@ -0,0 +1,228 @@
+# Long-Term Archiving
+
+Simulations that are deemed sufficiently valuable should be archived using `zstash` for long-term preservation. You can ask questions about `zstash` on the [zstash discussion board](https://github.com/E3SM-Project/zstash/discussions/categories/questions).
+
+!!! warning
+    Compy, Anvil and Chrysalis do not have local HPSS. We rely on NERSC HPSS for long-term archiving. If you are archiving a simulation run on Compy or LCRC (Chrysalis/Anvil), do all of the following steps. If you are archiving a simulation run on NERSC (Perlmutter), skip to step 4.
+
+## 1. Clean up directory
+
+Log into the machine that you ran the simulation on. Remove all `eam.i` files except the latest one. Dates are of the form `<YYYY-MM-DD>`.
+
+```shell
+cd <simulations_dir>/<case_name>/run
+ls | wc -l # See how many items are in this directory
+mv <case_name>.eam.i.<Latest YYYY-MM-DD>-00000.nc tmp.nc
+rm <case_name>.eam.i.*.nc
+mv tmp.nc <case_name>.eam.i.<Latest YYYY-MM-DD>-00000.nc
+ls | wc -l # See how many items are in this directory
+```
+
+There may still be more files than is necessary to archive. You can probably remove `*.err`, `*.lock`, `*debug_block*`, `*ocean_block_stats*` files.
+
+## 2. `zstash create` & Transfer to NERSC HPSS
+
+On the machine that you ran the simulation on:
+
+If you don’t have one already, create a directory for utilities, e.g., `utils/`. Then, open a file in that directory called `batch_zstash_create.bash` and paste the following in it, making relevant edits:
+
+```shell
+#!/bin/bash
+# Run on <machine name>
+# Load E3SM Unified
+<Command to load the E3SM Unified environment>
+# List of experiments to archive with zstash
+EXPS=(\
+<case_name> \
+)
+# Loop over simulations
+for EXP in "${EXPS[@]}"
+do
+    echo === Archiving ${EXP} ===
+    cd <simulations_dir>/${EXP}
+    mkdir -p zstash
+    stamp=`date +%Y%m%d`
+    time zstash create -v --hpss=globus://nersc/home/<first letter>/<username>/E3SMv2/${EXP} --maxsize 128 . 2>&1 | tee zstash/zstash_create_${stamp}.log
+done
+```
+
+Load the E3SM Unified environment.
+!!! tip
+    The E3SM Unified environment activation commands can be found on [zppy's Getting started page](https://e3sm-project.github.io/zppy/_build/html/main/getting_started.html). Alternatively, they can be found using [Mache](https://github.com/E3SM-Project/mache/tree/main/mache/machines): click the relevant machine and find the `base_path` listed under `[e3sm_unified]` -- the activation command will be `source <base_path>/load_latest_e3sm_unified_<machine_name>.sh`.
```

Comment:
We should update links to use `docs.e3sm.org` instead of `e3sm-project.github.io` now.